requirements/dev.in

```diff
@@ -0,0 +1,9 @@
+# Additional requirements for development of this application
+-c constraints.txt
+
+-r base.in                # Dependencies for edx-completion itself
+-r quality.in             # Dependencies for code quality checks
+-r test.in                # Dependencies for test case execution
+
+pip-tools                 # Requirements file management
+tox                       # virtualenv management for tests
```

Comment:
Please an empty line at the end of this file and also please add this dependency to a file called tox.in/tox.txt

Reply:
Added tox.in file. Added to ``make upgrade`` list of commands.